src/main/java/com/uc_it4045/assignment_tracker/AssignmentTrackerController.java

```diff
@@ -4,12 +4,12 @@
 
 @Controller
 public class AssignmentTrackerController {
-        /**
-         * Handle the / endpoint
-         * @return Start page template
-         */
-        @RequestMapping("/")
-        public String index() {
-            return "start";
-        }
+    /**
+     * Handle the / endpoint
+     * @return Start page template
+     */
+    @RequestMapping("/")
+    public String index() {
+        return "start";
     }
```

Comment:
Fixed line formatting. This is a best practice for readability if more code is added later